src/admin-views/tribe-options-general.php

```diff
@@ -112,20 +113,20 @@
 			'label'           => esc_html__( 'End of day cutoff', 'the-events-calendar' ),
 			'validation_type' => 'options',
 			'size'            => 'small',
-			'default'         => '12:00',
+			'default'         => date_i18n( $site_time_format, strtotime( '12:00 am' ) ),
 			'options'         => array(
-				'00:00' => '12:00 am',
-				'01:00' => '01:00 am',
-				'02:00' => '02:00 am',
-				'03:00' => '03:00 am',
-				'04:00' => '04:00 am',
-				'05:00' => '05:00 am',
-				'06:00' => '06:00 am',
-				'07:00' => '07:00 am',
-				'08:00' => '08:00 am',
-				'09:00' => '09:00 am',
-				'10:00' => '10:00 am',
-				'11:00' => '11:00 am',
+				'00:00' => date_i18n( $site_time_format, strtotime( '12:00 am' ) ),
```

Comment:
Shame we can't keep this in one place (tribe-common) and reuse via an array merge or something. Not a blocker though - we were already duplicating logic before this change.